static/scss/answers/interactive-map/InteractiveMap.scss

```diff
@@ -12,6 +12,8 @@
 
   $border-default: 1px solid #666 !default;
 
+  $locator-text-gray: #737373;
```

Comment:
Is this the color of all text in the locator? If so, I think we should name the variable `locator-text-color` or something like that.

Reply:
It's not the color all of the locator text (the locator has blue text for 'filter results' and 'update your location', and the location cards have black text), but it is the color used by all of the gray locator text